tests/conftest.py

```diff
@@ -40,6 +39,14 @@ def pytest_configure():
         PASSWORD_HASHERS=(
             'django.contrib.auth.hashers.MD5PasswordHasher',
         ),
+        **{
```

Comment:
1. I understand that `MIDDLEWARE` is the new setting name, and `MIDDLEWARE_CLASSES` is deprecated, but is it possible to have both of them in the settings without causing issues? That would avoid the need for doing the keyword argument unpacking.
2. I'm pretty sure keyword unpacking is only allowed at the end of the argument list, so that would explain the need to have it at the end.